### PR TITLE
Add Arabic and Spanish languages

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

File: config/sync/language.mappings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+_core:
+  default_config_hash: EMWe7Yu4Q5eD-NUfNuQAWGBvYUNZPIinztEtONSmsDc
+map:
```
Comment (amitaibu) [inline config/sync/language.mappings.yml:3]: @bboro Started for a clean installation, but the mapping is still there. I'll try to google..

Comment (amitaibu) [reply config/sync/language.mappings.yml:3]: I think it's just installed by default..

image

File: web/modules/custom/server_general/server_general.module [hunk 2]
```diff
@@ -41,3 +43,27 @@ function server_general_intl_date_formatted_date_alter(string &$formatted_date,
...
+
+/**
+ * Implements hook_field_widget_single_element_WIDGET_TYPE_form_alter().
+ *
+ * Language selection should be without the not defined and not applicable
+ * options.
+ *
+ * @see \Drupal\Core\Field\Plugin\Field\FieldWidget\LanguageSelectWidget::formElement()
+ */
+function server_general_field_widget_single_element_language_select_form_alter(array &$element, FormStateInterface $form_state, array $context) {
```
Comment (amitaibu) [inline web/modules/custom/server_general/server_general.module:67]: @bboro widget is now altered to not show the unwanted options.

File: web/modules/custom/server_general/server_general.module [hunk 3]
```diff
@@ -41,3 +43,27 @@ function server_general_intl_date_formatted_date_alter(string &$formatted_date,
...
+
+/**
+ * Implements hook_field_info_alter().
+ *
+ * Set default widgets.
+ */
+function server_general_field_info_alter(&$info) {
+  // Select2 for entity reference.
+  if (isset($info['entity_reference'])) {
+    $info['entity_reference']['default_widget'] = 'select2_entity_reference';
```
Comment (amitaibu) [inline web/modules/custom/server_general/server_general.module:55]: @dipakmdhrm ER's default widget is now select2

**Review by bboro — APPROVED** [review]

**Pull request merged**